src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Test
+	public void isAnyNull_noArgs() {
+		boolean result = Utils.isAnyNull();
+		assertEquals(false, result);
```

Comment:
You can also use `assertFalse` and put the argument to tell what is the problem, like #1199.